src/evaluate.js

```diff
@@ -14,82 +15,91 @@ export default function evaluate(tokens, expr, values) {
   for (var i = 0; i < numTokens; i++) {
     var item = tokens[i];
```

Comment:
question, wouldnt it be better to declare this vars on top and just assign here?

Reply:
Yes, it would be better. However, this is a fork of a 3rd party repo. In order to not prevent further merges from the base repo, I chose to not reformat the files too much.